utils/hare-shutdown

```diff
@@ -55,13 +55,14 @@ def _get_processes_for_node() -> Dict[str, List[utils.Process]]:
     return processes
 
 
-def stop_process(processes: Dict[str, List[utils.Process]]):
+def stop_process(processes: Dict[str, List[utils.Process]],
+                 skip_consul_stop: bool):
     for svc in utils.shutdown_sequence:
         if svc not in processes:
             continue
         if svc != 'consul':
             utils.stop_parallel([proc for proc in processes[svc]])
-        else:
+        elif skip_consul_stop == False:
```

Comment:
`eilf not skip_consul_stop:`

Reply:
Although I would prefer using 2 different [strategies](https://en.wikipedia.org/wiki/Strategy_pattern): one works as it used to, another one never stops consul.
The code of each strategy will be quite simple, the decision whether consul should be stopped or not is taken just once, at the very beginning. So lots of code will not need to propagate this `skip_consul_stop` flag and check it every now and then.

But again, if this is a draft and temporarily solution, please ignore this comment.